edenia/antelope-tools
# Feat/rebrand proton to xpr 1301

### Rebrand proton to XPR Network

### What does this PR do?

- Resolve #1301
- Refactor language json files to use the network name from the env variable

### Steps to test

1. Check that the changes in the .yaml files are correct
2. Check the sitemaps sitemap.xpr.xml and  sitemap.xpr-testnet.xml
3. Check the language json files
4. Run the project locally
5. Go to the homepage and check the change
6. Go to any other page and check the titles and descriptions

### Screenshots

# Homepage

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/c348043e-73ed-4dc3-9257-d6c35c4bbabe)

# Dashboard

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/6c10cf3e-2bbd-471b-99a7-4af307ae28a6)

# Block Producers page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/38b099b1-b9c3-4a08-ac26-624abfeb7c44)
